Helper_Scripts/benchmarks/email_search_bench.py

```
@@ -67,12 +67,19 @@
 ensure_repo_root()
 
 try:
-    from tldw_Server_API.app.core.DB_Management.Media_DB_v2 import MediaDatabase
+    from tldw_Server_API.app.core.DB_Management.media_db.api import create_media_database
 except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)